lib/webauthn/authenticator_data/attested_credential_data.rb

```diff
@@ -62,7 +62,7 @@ def length
       def valid_credential_public_key?
         cose_key = COSE::Key.deserialize(public_key)
 
-        !!cose_key.alg
+        !!cose_key.alg && WebAuthn.configuration.algorithms.include?(COSE::Algorithm.find(cose_key.alg).name)
```

Comment:
I've tried removing the right side of the `&&` and none of the existing test fail (nor the ones you had to add the algorightm to the configuration). Do you think it would be worth to right a test that fails in case the algorithm used in the key is not among the ones configured?

Reply:
Good call.

Thank you.